spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/JmsHealthIndicator.java

```diff
@@ -41,6 +46,12 @@ protected void doHealthCheck(Health.Builder builder) throws Exception {
 			builder.up().withDetail("provider",
 					connection.getMetaData().getJMSProviderName());
 		}
+		catch (Exception ex) {
+			logger.warn(String.format(
```

Comment:
I don't like that this is JMS-specific. Why not log something in `AbstractHealthIndicator`?